src/encoded/static/components/batch_download/cloning_mappings.js

```diff
@@ -0,0 +1,72 @@
+// libs
+import QueryString from '../../libs/query_string';
+import { uc } from '../../libs/constants';
+// local
+import BatchDownloadController, { buildDownloadOptionsIndex } from './base';
+
+
+/**
+ * Batch-download controller for downloads in individual `elements_cloning` and `elements_mappings`
+ * file tables.
+ * @param {object} dataset `elements_cloning` or `elements_mappings` dataset with files to download
+ */
+export default class CloningMappingsBatchDownloadController extends BatchDownloadController {
+    constructor(dataset) {
+        // Just need to reorder the parameters so `preBuildDownloadOptions` can access the
+        // parameters beyond `dataset` and `query`.
+        super(dataset, new QueryString());
+    }
+
+    preBuildDownloadOptions(dataset) {
+        this._hasAnalyses = !!(dataset.analyses?.length > 0);
+        this._downloadOptionsTemplate = [
+            {
+                id: 'all-files',
+                label: 'Download files',
+                title: `All experiment${this._hasAnalyses ? `${uc.rsquo}s analyses` : ''} files`,
```

Comment:
Should this be "all experiments' analysis files"?
![Screen Shot 2022-02-03 at 10 39 18 AM](https://user-images.githubusercontent.com/2887343/152407648-b66642a3-01e2-4569-a19d-6296cbb79acf.png)